core/store/migrate/migrations/0232_log_poller_word_topic_indexes.sql

```diff
@@ -0,0 +1,53 @@
+-- +goose Up
+
+drop index if exists evm.evm_logs_idx_data_word_one;
+drop index if exists evm.evm_logs_idx_data_word_two;
+drop index if exists evm.evm_logs_idx_data_word_three;
+drop index if exists evm.evm_logs_idx_data_word_four;
+drop index if exists evm.evm_logs_idx_topic_two;
+drop index if exists evm.evm_logs_idx_topic_three;
+drop index if exists evm.evm_logs_idx_topic_four;
+
+create index evm_logs_idx_data_word_three
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 65, 32));
+
+create index evm_logs_idx_data_word_five
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 129, 32));
+
+create index evm_logs_idx_topic_two
+    on evm.logs (evm_chain_id, address, event_sig, (topics[2]));
+
+create index evm_logs_idx_topic_three
+    on evm.logs (evm_chain_id, address, event_sig, (topics[3]));
+
+create index evm_logs_idx_topic_four
+    on evm.logs (evm_chain_id, address, event_sig, (topics[4]));
```

Comment:
What happened to data words one, two, and four.?

Reply:
I've removed them because data word queries are used only by CCIP and we rely only on 3 and 5

Reply:
If you feel it's too risky we can consider bringing them back. I just wanted to reduce number of redundant indexes from that table